### PR TITLE
Feature/add redux

### PR DESCRIPTION
**Pull request by bhskr44**
# Highlights

✔️ Add Reducers
✔️ Linters Fixed

### CODE REVIEW

File: src/redux/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuid4 } from 'uuid4';
+
+const initialState = [];
+
+const bookSlice = createSlice({
+  name: 'books',
```
Comment (FrederickMih) [inline src/redux/books.js]: - [ ] Hi @bhskr44, you're doing great in this project :+1: but I think we need to follow the project's requirements. We need to define [action types for adding(ADD_BOOK) and removing(REMOVE_BOOK) a book](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20action%20types%20for%20adding%20and%20removing%20a%20book.) and the value of each action type needs to follow the [duck pattern](https://github.com/erikras/ducks-modular-redux) :+1: 
- [ ] [Next](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=array%20of%20books.-,Export%20Action%20Creators%20for%20your%20actions.,-Write%20your%20reducer)

File: src/redux/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
```
Comment (FrederickMih) [inline src/redux/categories.js]: - [ ] Kindly check the comments above :arrow_up: . I think as developers, we shall always be required to follow strictly a client requirement. We really need to understand how to make our app scalable in the future using ducks pattern.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @bhskr44, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Great and descriptive README.md file :+1:
- PR is descriptive :+1: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

N/A

_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @bhskr44👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**